react-recaptcha-hook/src/use-recaptcha.tsx

```diff
@@ -1,14 +1,17 @@
 import * as React from 'react';
+
 import useScript from 'react-script-hook';
 
 export interface RecaptchaProps {
     sitekey: string;
     hideDefaultBadge?: boolean;
+    checkForExisting?: boolean;
 }
 
 export function useRecaptcha({
     sitekey,
     hideDefaultBadge = false,
+    checkForExisting = false,
```

Comment:
Why not as default behavior?
```javascript

checkForExisting = true

```
 I don't know any usecase where it would make sense to load the recaptcha script more than once

Reply:
By default I had put the same default value like useScript. But your are right.